Backend/Backend.WebServices/DataTransferObjects/MediaItemTypeDTO.cs

```diff
@@ -9,11 +10,11 @@ public MediaItemTypeDTO(MediaItemType mediaItemType)
         {
             Id = mediaItemType.Id;
             Name = mediaItemType.Name;
-            MediaItems = mediaItemType.MediaItems;
+            MediaItems = mediaItemType.MediaItems.Select(x => new MediaItemDTO(x)).ToList();
         } 
 
         public int Id { get; set; }
         public string Name { get; set; }
-        public virtual ICollection<MediaItem> MediaItems { get; set; }
+        public virtual ICollection<MediaItemDTO> MediaItems { get; set; }
```

Comment:
Hmm...should our MediaItemTypeDTO have a link back to MediaItems? So our object graph will be Enterprise -> Participant -> Place -> MediaItem -> MediaItemType -> MediaItem -> MediaItemType -> MediaItem...

The database object needs to have the relationship going both ways because of the foreign key, but our DTOs shouldn't have this two-way relationship. I think we can remove the MediaItems list from the MediaItemTypeDTO class.